examples/README.md

```diff
@@ -0,0 +1,26 @@
+# Quick Notes
+`dotenv` is listed as a dependencies in `package.json`.
+If you want to use this, add a `.env` file to the root of this project (same level as this `README.md` file)
+and replace the variables to fit your specific config.
+
+```
+BIGCHAINDB_API_PATH=http://localhost:9984/api/v1/
+BIGCHAINDB_APP_ID=<your-app-id>
+BIGCHAINDB_APP_KEY=<your-app-key>
```

Comment:
If you are running BigchainDB in you own environment i.e. localhost, then app id and app key are not needed and should be null. Maybe is better if you use instead the reference to BigchainDB testnetwok
```js
(
    "https://test.bigchaindb.com/api/v1/",
    {
        app_id: "Get one from testnet.bigchaindb.com",
        app_key: "Get one from testnet.bigchaindb.com"
    }
)
```

Reply:
What do you think @michielmulders ?

Reply:
+1 on this